static/js/popovers.js

```diff
@@ -372,6 +377,54 @@ export function hide_user_profile() {
     overlays.close_modal("#user-profile-modal");
 }
 
+function compare_by_name(a, b) {
+    return a.name.localeCompare(b.name);
+}
```

Comment:
We should use `util.strcmp` here; see the comments on `localeCompare` being really slow for why.  

Would you be up for doing a sweep?  It looks like `git grep localeCompare` finds a number of existing uses.